### PR TITLE
feat(content): More Snippets

### PR DESCRIPTION
**Pull request by Empyrean-Heaven**
## Purpose of change
Following up on my previous PR, this time I'll be adding more grave_labels, graffiti, and survivor_notes.

## Describe the solution
Adding more snippets.

## Describe alternatives you've considered
Leaving it be.

## Testing
JSON only.

## Additional context
Hope everyone lives a wonderful life. :D

### CODE REVIEW

**Comment by Empyrean-Heaven** [comment]
> The Autofix app has automatically formatted this Pull Request.
> 
> If you edit your PR on web UI, you can ignore this message. If you edit your PR locally, YOU MUST DO EITHER OF THE FOLLOWING:
> 
> * Run `git pull` to merge the automated commit into your local copy of the PR branch.
> * [Format your code locally](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style), and force push to your PR branch.
> 
> If you don't do this, your following work will be based on the old commit, and cause MERGE CONFLICT.

Sweet! Thanks Mr. autofix-ci!

File: data/json/snippets/graffiti.json [hunk 1]
```diff
@@ -110,7 +145,21 @@
...
-      { "id": "shelter_graffiti_35", "text": "Remember <given_name>" }
+      { "id": "shelter_graffiti_35", "text": "Remember <given_name>" },
+      {
+        "id": "shelter_graffiti_36",
+        "text": "Two homeless men were going at it while a zombie was eating a woman alive right next to them.  They even invited the zombie to join them."
+      },
+      {
+        "id": "shelter_graffiti_37",
+        "text": "A pregnant woman said that one of those undead fuckers fucked her in the fucking fuck fuck.  Fuck.  I had sloppy seconds.  It was worth it."
+      },
```
Comment (chaosvolt) [inline data/json/snippets/graffiti.json:156]: These two may lean a tad explicit admittedly.

File: data/json/snippets/survivor_notes.json [hunk 2]
```diff
@@ -584,6 +584,38 @@
...
+      },
+      {
+        "id": "note_226",
+        "text": "\"The taste of undead human meat is quite exquisite.  Scrumptious, and dare I say, morbidly delectable.  It is a much better alternative to regular human meat.\""
+      },
+      {
+        "id": "note_227",
+        "text": "\"I own a mininuke for home defense, since that's what the founding fathers intended.  Four ruffians break into my house.  'What the devil?'  As I grab my powdered wig and New England rifle.  Blow a golf ball sized hole through the first man, he's dead on the spot.  Draw my pistol on the second man, miss him entirely because it's smoothbore and nails the neighbor's dog.  I have to resort to the cannon mounted at the top of the stairs loaded with grape shot, 'Tally ho lads' the grape shot shreds two men in the blast, the sound and extra shrapnel set off car alarms.  Fix bayonet and charge the last terrified rapscallion.  He bleeds out waiting for the police to arrive since triangular bayonet wounds are impossible to stitch up.  Just as the founding fathers intended.\""
```
Comment (chaosvolt) [inline data/json/snippets/survivor_notes.json:598]: Excellent, though only the first line was changed to adapt the copypasta, and the rest is just the stanard version verbatim?

**Comment by Zireael07** [comment]
If those are the same snippets that are PR-d to CDDA (and from the pregnant woman one it looks like it is), then more snippets is good, middle school humor is not

**Comment by Empyrean-Heaven** [comment]
I'll remove ones that I added that are not my original writing. :D

**Review by chaosvolt — DISMISSED** [review]

File: data/json/snippets/survivor_notes.json [hunk 3]
```diff
@@ -584,6 +584,38 @@
...
+      },
+      {
+        "id": "note_232",
+        "text": "\"This is a reprint of an Imperial Decree issued by His Supremacy the Princeps of the Final Empire signed by the Princeps and His Excellency Chancellor Horatio Oswald.  By Imperial grace and the goodness of the Princeps and his Regent and Chancellor the noted contributions of one esteemed Charles E. Cheddar, brother of the acclaimed Chuck E. Cheese, has reached the ears of the populace and the Crown.  As issued in Chapter CXII of Scripture LVI of Novum Princeps IV the Lord Ruler granted every right over the dominion known as the Final Empire over to the Old Princes, which in turn was passed down to the Novum Princeps in Year 1829.  Henceforth Charles E. Cheddar, along with his kin Chuck E. Cheese, shall be granted the hereditary title of Baronet!  Congratulations,  Sir Charles and Sir Chuck.\""
```
Comment (chaosvolt) [inline data/json/snippets/survivor_notes.json:618]:
```suggestion
        "text": "\"This is a reprint of an Imperial Decree issued by His Supremacy the Princeps of the Final Empire signed by the Princeps and His Excellency Chancellor Horatio Oswald.  By Imperial grace and the goodness of the Princeps and his Regent and Chancellor the noted contributions of one esteemed Charles E. Cheddar, brother of the acclaimed Chuck E. Cheese, has reached the ears of the populace and the Crown.  As issued in Chapter CXII of Scripture LVI of Novum Princeps IV the Lord Ruler granted every right over the dominion known as the Final Empire over to the Old Princes, which in turn was passed down to the Novum Princeps in Year 1829.  Henceforth Charles E. Cheddar, along with his kin Chuck E. Cheese, shall be granted the hereditary title of Baronet!  Congratulations, Sir Charles and Sir Chuck.\""
```

**Review by chaosvolt — APPROVED** [review]
Okay, think it's done with formatting checks this time...

**Comment by RoyalFox2140** [comment]
> If those are the same snippets that are PR-d to CDDA (and from the pregnant woman one it looks like it is), then more snippets is good, middle school humor is not

Middle school humor I'll give it that, but a lot of real world graffiti is just foul language and insults when its not gang tags or complex images and fancy bubbly words. I think its very fitting if some of the graffiti was random profanity and the player is always free to scrub it off the walls since survivors writing toilet humor on the walls is a bit crude.

Also yes, I'm aware that the profane graffiti will be a high priority for locals to clean off the wall, but I'd say a lot of the madness between pre-cata and when the player is playing would lead to some of this graffiti being made but not cleaned up.

File: data/json/snippets/survivor_notes.json (hunk 2 above)
Comment (Empyrean-Heaven) [reply data/json/snippets/survivor_notes.json]: Edited to better fit. :D

**Pull request merged**